usrsctplib/netinet/sctputil.c

```diff
@@ -1686,58 +1732,33 @@ sctp_timeout_handler(void *t)
 		     (tmr->type != SCTP_TIMER_TYPE_SHUTDOWNACK) &&
 		     (tmr->type != SCTP_TIMER_TYPE_SHUTDOWNGUARD) &&
 		     (tmr->type != SCTP_TIMER_TYPE_ASOCKILL))) {
-			SCTP_INP_DECR_REF(inp);
```

Comment:
This decrement migrated to here as I understand:
https://github.com/sctplab/usrsctp/blob/4b77ee20ffa7a2b684f48e802f7b529a7151d978/usrsctplib/netinet/sctputil.c#L2584-L2598